crates/activate/src/lib.rs

```diff
@@ -524,24 +536,41 @@ mod test {
         ),)
     }
 
-    #[tokio::test]
-    async fn fixture_subtests() {
-        let source = build::arg_source_to_url("./src/test.flow.yaml", false).unwrap();
+    async fn managed_build(source: url::Url) -> build::Output {
```

Comment:
this was to get rid of build::managed_build ? Was it the last remaining usage?

This test introduced a pattern we've had in Go for a while, of doing an actual catalog build in service of establishing a test case fixture. We do want to keep that path as easy to walk as possible, perhaps by adjusting managed_build's interface if there are no other non-test usages.

Reply:
Yeah, this was to get rid of `build::managed_build`, as this was the last remaining usage. We're agreed that we want to keep an easy way to do builds of test fixtures. Perhaps it'll make sense to pull this back over into `build::managed_build` with a `#[cfg(test)]`, but there's also the option of using `Publisher::build` in tests. Both would have a few minor peculiarities, and I feel like I'd have a stronger opinion after seeing the needs of the tests, which is why I stuffed this in here for now.